events/bingo/serverScripts/bingoWheel/bingoWheelServer.js

```diff
@@ -31,13 +31,19 @@
     var currentRoundWinners = [];
     var BINGO_PRIZE_DOOR_1_TEXT = "{ff7674bb-5569-4381-b370-1dfa1d2a9723}";
     var BINGO_PRIZE_DOOR_1 = "{8724dc08-c0fb-4feb-bda1-8686023f8355}";
```

Comment:
Why are these hardcoded? I'm curious.

Reply:
they have to be for Bingo to work - there's no other way I can think of that would ensure that our content works reliably.